platforms/gke/base/use-cases/inference-ref-arch/terraform/deploy.sh

```diff
@@ -20,7 +20,6 @@ start_timestamp=$(date +%s)
 
 # Set use-case specific values
 export TF_VAR_initialize_backend_use_case_name="inference-ref-arch/terraform"
-export TF_VAR_resource_name_prefix="inf"
 
```

Comment:
Why was the prefix removed?

Reply:
Because if it's hardcoded, you can't change it without modifying the script.

Reply:
That is the point. 😀

Each use-case should have a unique prefix otherwise you could have collisions between use-cases/

Reply:
/shrug, I would have preferred to avoid hardcoding this to give users flexibility, but I'll revert this change.